ScanCommand/bal-scan-tool-tester/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+org = "ballerina"
+name = "bal_scan_tool_tester"
+version = "0.1.0"
+distribution = "2201.8.5"
```

Comment:
@Xenowa In the workflow and gradle script 2201.8.6 is used. Shall we stick to that?

Reply:
Scan tool tests have been refactored to work similar to tests of other Ballerina commands in [691499](https://github.com/ballerina-platform/static-code-analysis-tool/pull/3/commits/691499940e4f1889ce3b1b15e0c5abe412a4c2c1)